### PR TITLE
blocking the "Continue" button after it has been clicked once.

### PR DESCRIPTION
**Pull request by VishalPraneeth**
Close: #7056 

## PR Details

Disable the usage of the continue button while submission as mentioned in the issue.

Issue Details

Currently, the "Continue" button in the application's wizard allows multiple clicks, which can lead to unintended consequences or user confusion. This issue aims to improve the user experience by implementing a feature to block the "Continue" button after it has been clicked once.

Proposed Solution

Implement a mechanism to disable the "Continue" button after it has been clicked to prevent multiple submissions.
Ensure that the button is visually disabled after the first click to provide clear feedback to users.
Consider adding a loading indicator to the button during form submission to further indicate that the action is being processed.

### Summary
🚀🛠️🚫

Improve create service wizard UX and performance. Prevent duplicate actions and disable `continue` button when appropriate in `CreateServiceWizard.tsx`.

> _Create service fast_
> _`continueButtonClicked`_
> _Autumn leaves no dupes_

### Walkthrough
*  Prevent duplicate analytics events and API calls when user clicks continue button on wizard pages ([link](https://github.com/amplication/amplication/pull/7192/files?diff=unified&w=0#diff-5a8f852ecd706b64aada842f7e069be776805096dcc064c8172b93d0388e14cdR370-R371), [link](https://github.com/amplication/amplication/pull/7192/files?diff=unified&w=0#diff-5a8f852ecd706b64aada842f7e069be776805096dcc064c8172b93d0388e14cdL378-R392), [link](https://github.com/amplication/amplication/pull/7192/files?diff=unified&w=0#diff-5a8f852ecd706b64aada842f7e069be776805096dcc064c8172b93d0388e14cdR540-R545))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by souravjain540** [comment]
please never make a pr without getting it assigned to yourself. please comment on the issue to get it assigned.

**Comment by VishalPraneeth** [comment]
> please never make a pr without getting it assigned to yourself. please comment on the issue to get it assigned.

Yea, my bad! I've commented on the issue to get it assigned.

File: packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx [hunk 1]
```diff
@@ -528,6 +535,12 @@ const CreateServiceWizard: React.FC<Props> = ({
           trackWizardPageEvent={trackWizardPageEvent}
           {...serviceNextStepStatus}
         />
+        <button
+          type="submit"
+          disabled={continueButtonClicked || loadingCreateService}
+        >
+          Continue
+        </button>
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx:543]: Isn't this button duplicate? 
what will happen with the button is currently shown from the WizardFooter?
Please add a screenshot or a video of the solution before we continue the review.

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx (hunk 1 above)
Comment (VishalPraneeth) [reply packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx:543]: yep it creates a duplicate at the WizardFooter, I guess I'll have to make changes in the CreateServiceWizardFooter.tsx

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**